c2rust/src/main.rs

```diff
@@ -7,7 +7,7 @@ use std::process::{exit, Command};
 git_testament!(TESTAMENT);
 
 fn main() {
-    let subcommand_yamls = [load_yaml!("transpile.yaml"), load_yaml!("instrument.yaml")];
```

Comment:
The `c2rust instrument` binary now lives in `c2rust-instrument` rather than `c2rust`.  Shoehorning it back into `c2rust` is very unwieldy and would need extra code to do.  The deprecated clap YAML way is not used anymore, so it's more difficult.

Reply:
If `c2rust/arc/main.rs` didn't do the argument checks itself, it would still work, as would `c2rust-analyze` and `c2rust-pdg`.

Reply:
I do think the subcommand way needs improving, so I didn't think extra code was worth it here if it's not going to be eventually used, and it would further complicate this PR.

Reply:
Sounds good